docs/dependencies.rst

```diff
@@ -0,0 +1,12 @@
+Dependencies
+------------
+Conda-press currently has the following external dependencies,
+
+*Run Time:*
+
+#. xonsh
+#. lazyasd
+#. ruamel.yaml
+#. tqdm
+#. requests
+#. conda (installed in the current environment)
```

Comment:
"All of these dependencies are available in `conda-forge`."

?

Reply:
Sorry, but what do you mean? Do you think would be helpful to add that sentence or you are actually asking? But all those dependencies are in ``conda-forge`` indeed.